src/tests/adding.md

```diff
@@ -500,3 +500,20 @@ Besides `normalize-stderr-32bit` and `-64bit`, one may use any target
 information or stage supported by [`ignore-X`](#ignoring-tests) here as well (e.g.
 `normalize-stderr-windows` or simply `normalize-stderr-test` for unconditional
 replacement).
+
+## Input Normalization
+
+Sometimes, you want to normalize the inputs to a test. For example, you may
+want to pass `// compile-flags: --x=y.rs`, where y.rs is some file in the test
+directory. In this case you can use input normalization. The following strings
+are replaced in header inputs:
+
+- {{cwd}}: The directory where compiletest is run from. This may not be the
+  root of the checkout, so you should avoid using it where possible.
+  - Example: `/path/to/rust`
```

Comment:
```suggestion
  - Examples: `/path/to/rust`, `/path/to/build/root`
```
This makes it a bit clearer that it doesn't have to be the root of the checkout.